msgraph-sdk-raptor-compiler-lib/appsettings.json

```
@@ -4,5 +4,8 @@
   "DocsRepoCheckoutDirectory": "C:/github",
```

not this PR, but you'll probably want to clean up windows specific paths.

---

it was just a convenience for myself and we were replacing this in linux runs. But I might as well remove it.
```suggestion
  "DocsRepoCheckoutDirectory": "",
```